source/libtomo/gridrec/tables.h

```diff
@@ -50,11 +50,9 @@
 // Use X/Open-7, where posix_memalign is introduced
 #define _XOPEN_SOURCE 700
 
-#include "filters.h"
 #include "mkl.h"
```

Comment:
Why does tables.h need mkl? Double check all these gridrec module headers. I think only gridrec.c and gridrec.cc need mkl.